src/qt/utilitydialog.cpp

```diff
@@ -22,7 +22,7 @@
 #include <QCloseEvent>
 #include <QLabel>
 #include <QMainWindow>
-#include <QRegExp>
+#include <QRegularExpression>
```

Comment:
Add `#include <QString>` as well (due to using `QStringLiteral`)?

Reply:
Done in https://github.com/bitcoin-core/gui/pull/620/commits/ace9af5688662a56d855bd26f36bc5c145b909e3.